src/collections/unordered-map.ts

```diff
@@ -59,21 +59,21 @@ export class UnorderedMap {
     return keysIsEmpty;
   }
 
-  get(key: Bytes): unknown | null {
+  get(key: Bytes, deserializer?: (value: unknown) => DataType): DataType | null {
```

Comment:
@petarvujovic98 since I see that you are working on the default value, maybe it would be good to convert this to options

Reply:
@osalkanovic Seems reasonable to me. This also allows for more extensibility in the future